Gemfile.lock

```diff
@@ -38,24 +38,23 @@ GEM
       i18n (~> 0.7)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-    addressable (2.5.1)
-      public_suffix (~> 2.0, >= 2.0.2)
-    algoliasearch (1.14.0)
+    addressable (2.4.0)
+    algoliasearch (1.16.0)
       httpclient (~> 2.8.3)
       json (>= 1.5.1)
     algoliasearch-rails (1.14.1)
       algoliasearch (>= 1.2.14)
       json (>= 1.5.1)
     arel (8.0.0)
     ast (2.3.0)
-    backports (3.8.0)
+    backports (3.9.1)
```

Comment:
Please ensure `bundle install` only affects the algolia-specific dependencies in this branch.